scripts/info.c

```diff
@@ -9,6 +9,9 @@ void uftrace_begin(struct script_info *sc_info)
 	printf("%s\n", sc_info->version);
 	printf("%s\n", sc_info->name);
 	printf("%s\n", sc_info->cmds);
+
+	/* disable actual data record by uftrace */
+	sc_info->record = false;
```

Comment:
As I said before, I'd like to have two bitmasks to control the behavior.

Reply:
I can make it to use two bitmasks, but I don't see if it's useful to control argument recording so left it for now. I can update it in the next update.

Reply:
Let's do this in a separate PR.  I think it's better to focus on enabling the native script here.  Also other scripting like Python might use this feature too.